modules/common/react/lib/theming/styled.ts

```diff
@@ -28,9 +28,9 @@ export const convertToStaticStates = (obj?: CSSObject): CSSObject | undefined =>
 function styled<Props>(node: any) {
   return (...args: Interpolation<Props>[]) => {
     const newArgs: Interpolations = args.map(
-      interpolation => (props: Props & {theme: CanvasTheme & {_staticStates?: boolean}}) => {
+      interpolation => (props: Props & {theme: EmotionCanvasTheme & {_staticStates?: boolean}}) => {
```

Comment:
Now that we've updated the theme to be namespaced under `canvas`, we obviously have to add the extra key every time we access a theme value (i.e. `theme.canvas.palette.primary.main`).

I tried to do some magic so that our components were only passed the `canvas` object for their theme. This function started to get quite messy (since it could be called with `theme.canvas.palette` or `theme.palette`), and it became confusing to know which version of the theme you were using in certain places. This is a bit more verbose, but I think it's clearer when there's less magic going on.

Would like some input on this.